README.md

```diff
@@ -4,3 +4,104 @@ Crafter Studio
 Crafter Studio is the authoring environment for Crafter CMS. For more information, please read the [docs.](http://docs.craftercms.org/en/latest/developers/projects/studio/index.html)
 
 We welcome community contributions, please read [CONTRIBUTING.md](https://github.com/craftercms/studio/blob/master/CONTRIBUTING.md) for guidelines.
+
+
+## Using the Upgrade System
+
+When a significant change is made in the way that Studio works the PR should include the required upgrades to make sure that sites created in previous versions will continue to work. The upgrade system provides four diferent pipelines.
+
+All pipelines are configured in the same [file](https://github.com/craftercms/studio/blob/develop/src/main/resources/crafter/studio/upgrade/pipelines.yaml).
+
+### Blueprints
+
+In the current version of the upgrade system any change made to the blueprints in this [repository](https://github.com/craftercms/studio/tree/develop/src/main/webapp/repo-bootstrap/global/blueprints) will be automatically copied to the local global repository when Studio boots.
+
+*In the future this will change to follow the same approach used for the other components.*
+
+### Database & Configuration
+
+When there is a change in the database (structure or conent) there are two requirements:
+
+1. Update the create [script](https://github.com/craftercms/studio/blob/develop/src/main/resources/crafter/studio/database/createDDL.sql)
+2. Add an upgrade script in the [database folder](https://github.com/craftercms/studio/tree/develop/src/main/resources/crafter/studio/database)
```

Comment:
Can we add a little more description here that the upgrade script is for the database?  It would also be nice if we have some sort of sample database upgrade script?